core/scripts/sql/texera_ddl.sql

```diff
@@ -19,6 +20,14 @@ CREATE TABLE IF NOT EXISTS user
 ) ENGINE = INNODB,
 -- start auto increment userID from 1 because userID 0 means user not exists
   AUTO_INCREMENT = 1;
+  
+CREATE TABLE `user_dictionary` (
+  `uid` int unsigned NOT NULL,
+  `key` varchar(255) COLLATE utf8mb4_bin NOT NULL,
```

Comment:
why are we using 255, if possible use 256 is better?


also is the `utf8mb4_bin` encoding necessary? if not please remove it.

Reply:
hmm, Seems to work without utf8mb4 encoding.

As for why varchar(255), I thought that with 255 characters, the varchar length will fit in 1 byte.
However, I just realized that with utf-8 encoding, not every character is 1 byte, so 255 characters could be more than 255 bytes long, so that wouldn't work anyways